manager/misc.go

```diff
@@ -33,18 +33,22 @@ const (
 type BundleErrorMessage string
 
 const (
-	BundleMkdirFailed = BundleErrorMessage("Failed to create bundle file directory")
-	BundleZipFailed   = BundleErrorMessage("Failed to compress the support bundle files")
-	BundleOpenFailed  = BundleErrorMessage("Failed to open the compressed bundle file")
-	BundleStatFailed  = BundleErrorMessage("Failed to compute the size of the compressed bundle file")
+	BundleMkdirFailed               = BundleErrorMessage("Failed to create bundle file directory")
+	BundleZipFailed                 = BundleErrorMessage("Failed to compress the support bundle files")
+	BundleOpenFailed                = BundleErrorMessage("Failed to open the compressed bundle file")
+	BundleStatFailed                = BundleErrorMessage("Failed to compute the size of the compressed bundle file")
+	BundleDownloadPercentageForLogs = 80
```

Comment:
`BundleProgressPercentageYAMLs`
`BundleProgressPercentageLogs`
`BundleProgressPercentageTotal`

Put yamls first because it's the first generated.